avocado/__init__.py

```diff
@@ -35,3 +38,5 @@
 from avocado.core.exceptions import TestError
 from avocado.core.exceptions import TestFail
 from avocado.core.exceptions import TestCancel
+from avocado.core.output import LOG_JOB
+from avocado.core.output import LOG_UI
```

Comment:
Ok, this makes a lot of sense. I see you're exposing the new API and also changing the modules to consume it. But you're still missing some modules (`optional_plugins`, `examples/plugins/...`). I'd suggest to split this commit, keeping here only the new API creation, adding a new commit that includes the modules updates, including the missing modules.